NGCHM/WebContent/javascript/PaneControl.js

```diff
@@ -659,6 +662,67 @@
     return pane;
   }
 
+  // Exported function.
+  // Set the heatMap-specific decorative UI elements.
+  PANE.setPaneDecor = setPaneDecor;
+  function setPaneDecor (loc, decor) {
+    // Remove previous custom decor.
+    loc.paneHeader.style.backgroundColor = "";
+    const av = loc.paneHeader.getElementsByClassName("paneAvatar")[0];
+    while (av.firstChild) av.firstChild.remove();
+    if (!decor) {
+      // Quit if no custom decor.
+      return;
+    }
+    if (decor.hasOwnProperty('hue')) {
+      const bgColor = UTIL.hsvToRgb (decor.hue, 10, 90);
```

Comment:
I like these choices for saturation and value, and how the resulting colors match with the avatars.